package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-practical-7",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "bcryptjs": "^2.4.3",
```

Comment:
It would be best if you use the build-in library `crypto` for encryption and decryption